tests/test_cli/test_add/test_connection.py

```diff
@@ -64,6 +64,10 @@ def setup_class(cls):
         shutil.copytree(Path(CUR_PATH, "..", "packages"), Path(cls.t, "packages"))
 
         os.chdir(cls.t)
+        result = cls.runner.invoke(
+            cli, [*CLI_LOG_OPTION, "init", "--author", AUTHOR], standalone_mode=False
```

Comment:
We are not resetting this. Ideally, one would clean up after each test...but ideally we would not have to run this in the first place.